_data/simple-icons.json

```diff
@@ -11647,6 +11647,11 @@
             "source": "https://policies.pixiv.net/en.html#brand",
             "guidelines": "https://policies.pixiv.net/en.html#brand"
         },
+        {
+            "title": "Pixlr",
+            "hex": "7286FF",
```

Comment:
Colors are changing while scrolling and changes from page to page. I wonder if we should go with one the colors mentioned in https://blog.pixlr.com/pixlr-reimagined-new-logo-new-look-new-features/?

Reply:
Of those 3 colours, I think we should run with the blue. Not at a computer again 'til Monday so feel free to push the change in the meantime.